ambassador/ambassador/config/acmapping.py

```diff
@@ -23,7 +23,7 @@
 ## Each Mapping object has a group_id that reflects the group of Mappings
 ## that it is a part of. By definition, two Mappings with the same group_id
 ## are reflecting a single mapped resource that's going to multiple services.
-## This implies that Mapping.group_id() is a very, very, very important 
+## This implies that Mapping.group_id() is a very, very, very important
```

Comment:
nit: I see that this kind of comments is all over Ambassador, but try to avoid subjective language as much as possible.

Reply:
Agreed, although I didn't add this comment; my IDE just decided to remove the whitespace when I saved. Happy to update this comment for clarity while I'm in here if you'd like though.

Reply:
I wrote it. :smile: Happy to entertain suggestions as a separate PR. :smile:

Reply:
@patricksanders Thanks, We don't need to change anything here.